protoc-gen-swagger/genswagger/template.go

```diff
@@ -387,13 +389,17 @@ func renderServices(services []*descriptor.Service, paths swaggerPathsObject, re
 				}
 
 				methProtoPath := protoPathIndex(reflect.TypeOf((*pbdescriptor.ServiceDescriptorProto)(nil)), "Method")
+				desc := ""
+				if meth.GetServerStreaming() {
+					desc += "(streaming responses)"
```

Comment:
@achew22 swagger spec doesn't address streaming endpoints (yet) there's some discussion on the openapi repo.